### ISSUE TITLE
The Maven Repo does't work

### ISSUE THREAD

**Issue by mlh1100** [TheHolyWaffle/TeamSpeak-3-Java-API#171]
I copy&pasted the dependeny and repo but im getting an error that the version does not exists!
Does it still exists or is it broke.

**Comment by somefriggnidiot** [comment]
Try using just the dependency. I've had the same issue, but my project builds fine without the repo.

**Comment by rogermb** [comment]
I just tested it locally and - on my machine at least - everything worked perfectly:

I deleted the the cached entries from my local repository (the .m2 folder in your user directory on Windows), created a new project, added the repo and dependency description to the project's pom.xml, told Maven to re-import the project and it correctly imported the newest 1.0.14 snapshot build from the Frankenstein's GitHub Repository.

Could it be that you're using an ancient version of Maven? Maybe Maven 2 even? Or is there anything else wrong with your pom.xml file?

This issue is quite important to me, as this repository *really* needs to work properly for everyone.
